bulma/components/tooltip.sass

```diff
@@ -0,0 +1,31 @@
+.tooltip
+  position: relative
```

Comment:
What if the element is already positioned (through another class) or has `overflow: hidden`?